src/thor/route_action.cc

```diff
@@ -232,6 +232,7 @@ thor::PathAlgorithm* thor_worker_t::get_path_algorithm(const std::string& routet
                                                        const valhalla::Location& destination) {
   // Have to use multimodal for transit based routing
   if (routetype == "multimodal" || routetype == "transit") {
+    valhalla::midgard::logging::Log("algorithm::multimodal ", " [ANALYTICS] ");
```

Comment:
it seemed crazy to me that we dont log the algorithm that we use. in fact this was the source of much frustration for me. i wanted to show that this implementation speeds up timedep forward but everytime i tried a route from say minsk to london it was the same speed as bidir astar and the reason for that was...